tests/phpunit/includes/TestCase.php

```diff
@@ -40,7 +41,7 @@ public static function tearDownAfterClass() {
 	}
 
 	protected static function reset_feature_flags() {
-		Feature_Flags::set_mode( Feature_Flags::MODE_PRODUCTION );
+		Build_Mode::set_mode( Build_Mode::MODE_PRODUCTION );
```

Comment:
Let's not touch the build mode here since this method is specific to feature flags. It's probably a good idea to reset this like we do with features, but it should happen separately now since we're decoupling these things.

Reply:
Updated.